packages/ui-components/src/styles/errorPages.css

```diff
@@ -0,0 +1,5 @@
+@import "./_colors";
+
+svg > .svg_primaryColor {
```

Comment:
we need to contain all css under a specific library id (example #microlc svg >....), this is done during the bundle with rollup?

Reply:
it is done automatically by rollup with postcss